01-HTML-CSS-Basics/1. HTML/Punto 2/v. Sign Up Validation.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Sign Up</title>
+  <meta name="Descripcion" content="Topic-1 SignUp de persona">
```

Comment:
Meta tags name should be in english and preferably exsiting ones: http://www.tutorialspoint.com/html/html_meta_tags.htm